webhooks-extension/test/setup_webhook_simple_test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+export tekton_repo_dir=$(git rev-parse --show-toplevel)
+export test_dir="${tekton_repo_dir}/webhooks-extension/test"
+
+source ${test_dir}/config.sh
+source ${test_dir}/util.sh
+
+# This script should be run after install_prereqs.sh and install_dashboard_and_extension.sh
+# 
+# Set up a webhook to trigger a pipeline suitable for manual testing via git push. 
+# All secrets and tekton resources will be created in the same namespace as the dashboard for the purpose of this test. 
+
+# Create secrets
+# 1. We need a host:ip for the dashboard to curl commands to
+#    Typically we port-forward the dashboard, but in this first version we're going to create a NodePort service
+#    and talk to that via localhost:31001. 
+
+check GOPATH
+if [ ! -f $GOPATH/src/github.com/tektoncd/experimental/webhooks-extension/test/credentials.sh ]; then 
+  echo "${GOPATH}/src/github.com/tektoncd/experimental/webhooks-extension/test must exist and contain adequate config. "
+  exit 1
+fi
+pushd $GOPATH/src/github.com/tektoncd/experimental/webhooks-extension/test
+. ./credentials.sh
+
+kubectl apply -f dashboard-service.yaml
```

Comment:
For this and the other 3 apply, should we add `-n $DASHBOARD_INSTALL_NS`?

Reply:
yes, definitely - thank you